.gitlab-ci.yml

```diff
@@ -9,7 +9,7 @@ stages:
   - github-release
 
 .go-cache:
-  image: 'docker.repo.splunkdev.net/ci-cd/ci-container:golang-1.16'
+  image: 'docker.repo.splunkdev.net/ci-cd/ci-container:golang-1.17'
```

Comment:
The `golang-1.17` tag currently doesn't exist.  But there is `golang-1.17-pre`, which could possibly change.